scripts/gen_architecture_overview.py

```
@@ -0,0 +1,175 @@
+# ---- facade + config roles (no wires -- position carries the hierarchy) --
+facade_cx = (core_l + core_r) / 2
+box(facade_cx, Y_FACADE, "SolidSyslog", YELLOW, YELLOW_E, link=FACADE_LINK, bold=True, w=184, h=50)
+for cfg, ccx in config:
+    box(ccx, Y_CONFIG, cfg, GREEN, GREEN_E, link=f'../../api/structSolidSyslog{cfg}/')
+
+# ---- touch row: roles crossing the Core boundary to your system ----------
+for t in touch:
+    box(t["cx"], Y_TOUCH, t["name"], GREEN, GREEN_E, link=f'../../api/structSolidSyslog{t["name"]}/')
+    box_bottom = Y_TOUCH + BOX_H
+    mid = (box_bottom + band_top) / 2
+    for cx2, ext in zip(t["slots"], t["exts"]):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)